examples/pytorch/question-answering/fine-tune-distilbert.ipynb

```diff
@@ -0,0 +1,589 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b0ee5aff-46d6-47c5-8306-901cfd9206d9",
+   "metadata": {},
+   "source": [
+    "# Fine-tuning DistilBERT for question answering\n",
+    "\n",
+    "This guide describes fine-tuning DistilBERT with Stanford Question Answering Dataset (SQuAD) for question-answering using Kubeflow Trainer.\n",
+    "\n",
+    "This guide is adapted from HuggingFace question answering task recipe page: https://huggingface.co/docs/transformers/en/tasks/question_answering\n",
+    "\n",
+    "Pretrained DistilBERT: https://huggingface.co/docs/transformers/en/model_doc/distilbert\n",
+    "\n",
+    "SQuAD dataset: https://huggingface.co/datasets/rajpurkar/squad"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c31bc8f2",
+   "metadata": {},
+   "source": [
+    "# Install the KubeFlow SDK and dependencies"
```

Comment:
Let's remove the installation of Kubeflow SDK as prerequisite for this Notebook.
It is better to redirect users to this guide to get started with Kubeflow Trainer: 
https://www.kubeflow.org/docs/components/trainer/getting-started/